ctl_main.c

```diff
@@ -1238,6 +1238,58 @@ static int cmd_showport(int argc, char *const *argv)
     return r;
 }
 
+static int cmd_showportparams(int argc, char *const *argv)
+{
+    int r = 0;
+
+    int br_index = get_index(argv[1], "bridge");
+    if(0 > br_index)
+        return br_index;
+
+    int i, count = argc - 3;
+    param_id_t param_id[count];
+
+    for(i = 0; i < count; ++i)
+    {
+        int err = 1;
+        for(int j = 0; j < COUNT_OF(cist_port_params); ++j)
+        {
+            if(0 == strcmp(argv[argc - count + i], cist_port_params[j].str))
+            {
+                param_id[i] = cist_port_params[j].id;
+                err = 0; break;
```

Comment:
nitpick: `break` on next line

Reply:
thank you, fix in https://github.com/mstpd/mstpd/pull/132/commits/c52ed4ff948c2a44abfc6193ef0f129c366047e1